script.js

```diff
@@ -0,0 +1,61 @@
+var name = prompt("Hi there, what's your name?");
+var year = prompt("Nice to meet you "+name+", what year would you like to go to?");
+
+for(var i = year; i <= 2015; i++) {
+name = "Calvin Klein";
+}
+
+if(year >= 2015) {
+    console.log("I see you're a fan of Back To The Future 2, would you rather deal with Biff or Griff?");
+    var biffOrGriff = prompt("Biff or Griff? (Please enter 'b' or 'g')");
+      if(biffOrGriff === "b") {
+        console.log("Hmm, interesting. Biff is angry and has a cane. Do you stand and fight, or run away like a coward?");
+        var standOrRun = prompt("Fight or Run? (Please enter 'f' or 'r')");
+          if(standOrRun === "f") {
+            console.log("Good choice. Biff is old and feeble at this point. You push him over and he falls in a pile of manure.");
+          } else {
+            console.log("You get away, but your future son Marty Jr. is heckled for the rest of his days for his dad's cowardice.");
+          }
+    } else {
+        console.log("Griff is asking you if you are in, or out. What do you say?");
+        var inOrOut = prompt("In or Out? (Please enter 'i' or 'o')");
+          if(inOrOut === 'i') {
+            console.log("Bad call. Griff and his cronies rob the Hill Valley bank and frame you for it. No more time travel for you.");
+          } else {
+            console.log("Good call. You deck Griff in the jaw and run away. He gives chase on his hoverboard and ends up in a pile of manure.");
+          }
+      }
+}
+//I think this section would be best for a loop but I'm not sure how to incorporate it in a meaningful way.
+else if(year >= 1985 && year <= 2014) {
```

Comment:
Right now, this code is technically fine, in that I would use an if statement for the 1985 - 2014 test, but I could see incorporating a loop *inside* this section, which maybe increments year-by-year and gives you updates or something.